### ISSUE TITLE
Vermilion Gym Guide mentions Bird type instead of Flying type

### ISSUE THREAD

**Issue by anoomolu** [Deokishisu/FRLG-Plus#43]
Hiya. Been having a blast playing through FRLG+ so far.

I noticed that the gym guide in Vermilion mentions Bird type as being weak to Electric rather than Flying - unsure if this is also in the vanilla game but it seemed kinda odd.

![image](https://github.com/user-attachments/assets/6bbc5160-6b86-4cef-97b7-fdce6ab9691e)

**Comment by Deokishisu** [comment]
Good catch. This is actually present in vanilla (and in the original RBY text from this NPC). It will be fixed in the next release.
